src/graphql/queries.js

```diff
@@ -1,5 +1,8 @@
 // Code related to GraphQL queries, fragments, variables, etc.
 
+// IMPORTANT: queries here may be used in the offline mode to create mock data. Before removing or renaming
+// queries here, please check under the services/mock folder for occurrences of the variable name.
+
```

Comment:
Just a note. FWIW, the offline mode is broken because the query in `get-checkpoint.py` was not excluding the `root` family, which was an improvement from another recent PR.